app/src/main/java/com/github/factotum_sdp/factotum/MainActivity.kt

```diff
@@ -34,7 +36,9 @@ class MainActivity : AppCompatActivity() {
 
         val drawerLayout: DrawerLayout = binding.drawerLayout
         val navView: NavigationView = binding.navView
-        val navController = findNavController(R.id.nav_host_fragment_content_main)
```

Comment:
I don't get why it's a refactor here ? It seems that these two new lines does the job of one.

Reply:
It's the way to have a navController when you have a FragmentContainerView